python/tvm/relay/frontend/tflite.py

```diff
@@ -2710,6 +2743,145 @@ def convert_unpack(self, op):
 
         return squeezed
 
+    def convert_unidirectional_sequence_lstm(self, op):
+        """Long Short Term Memory for TFLite implementation."""
+        if self.is_quantized(op):
+            raise tvm.error.OpNotImplemented(
+                "TFlite quantized UNIDIRECTIONALSEQUENCELSTM operator is not supported yet."
+            )
+
+        input_tensors = self.get_input_tensors(op)
+        assert len(input_tensors) >= 20, "input tensors length should be >= 20"
```

Comment:
can this LSTM function also support any small input tensor case? for example input tensor size = 1?

Reply:
As I mentioned above, from our understanding about tflite format, the number of vectors which can be stored in flatbuffer designed by tflite is 20 tensors. If you generate a tflite model without specifying other weights except input tensor, the ignored weights will be assigned by default values (0) and store in model file as inputs. Of course, in this code we just support some recent tflite versions. If you can find some models which support different numbers of inputs and the references from tflite documentation to instruct how to process it, we deeply thank you and extend out support for that kind of model.